cronet/src/main/java/io/grpc/cronet/CronetChannelBuilder.java

```diff
@@ -173,7 +173,7 @@ final CronetChannelBuilder setTrafficStatsTag(int tag) {
    * Sets specific UID to use when accounting socket traffic caused by this channel. See {@link
    * android.net.TrafficStats} for more information. Designed for use when performing an operation
    * on behalf of another application. Caller must hold {@link
-   * android.Manifest.permission#MODIFY_NETWORK_ACCOUNTING} permission. By default traffic is
+   * android.Manifest.permission#UPDATE_DEVICE_STATS} permission. By default traffic is
```

Comment:
https://android.googlesource.com/platform/frameworks/base/+/master/core/res/AndroidManifest.xml#6416

MODIFY_NETWORK_ACCOUNTING is deprecated and recommendation is to use UPDATE_DEVICE_STATS